### PR TITLE
[3.1.x] Handle unreachable remotes

### PR DESCRIPTION
**Pull request by rart**

### CODE REVIEW

File: ui/app/src/translations/locales/en.json [hunk 1]
```diff
@@ -295,10 +295,10 @@
   "publishingDashboard.confirmAllHelper": "Set the state for all selected items to \"Cancelled\"?",
   "publishingDashboard.confirmHelper": "Set the state for all selected items to \"Cancelled\"",
   "publishingDashboard.confirmHelperText": "Set item state to \"Cancelled\"?",
-  "publishingDashboard.environment": "Publishing Target",
+  "publishingDashboard.environment": "Publishing",
   "publishingDashboard.fetchPackagesFiles": "Fetch Packages Files",
   "publishingDashboard.filesList": "files list",
-  "publishingDashboard.filteredBy": "Showing: {state, select, all {} other {Status: {state}.}} {environment, select, all {} other {{environment} target.}} {path, select, none {} other {Filtered by {path}}}",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/en.json]: Why are we rolling back these changes? We want to remove the word environment from this context and make it target/publishing target.

File: ui/app/src/translations/locales/en.json [hunk 2]
```diff
@@ -310,16 +310,18 @@
   "publishingDashboard.scheduled": "Scheduled for <b>{schedule, date, medium} {schedule, time, short}</b> by <b>{approver}</b>",
   "publishingDashboard.selectAll": "Select all on this page",
   "publishingDashboard.state": "State",
-  "publishingDashboard.status": "Status is {state} for {environment} publishing target",
+  "publishingDashboard.status": "Status is {state} for {environment} environment",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/en.json]: This is the same issue.

File: ui/app/src/translations/locales/en.json [hunk 3]
```diff
@@ -295,10 +295,10 @@
   "publishingDashboard.confirmAllHelper": "Set the state for all selected items to \"Cancelled\"?",
   "publishingDashboard.confirmHelper": "Set the state for all selected items to \"Cancelled\"",
   "publishingDashboard.confirmHelperText": "Set item state to \"Cancelled\"?",
-  "publishingDashboard.environment": "Publishing Target",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/en.json]: Why remove the word target?

File: ui/app/src/translations/locales/de.json [hunk 4]
```diff
@@ -313,13 +313,15 @@
...
   "reposAdmin.pendingCommit": "Repo enthält Dateien, für die ein Commit aussteht. Weitere Informationen finden Sie unter Repository-Status.",
+  "reposAdmin.unreachableRemote": "Remote \"{name}\" ist derzeit nicht erreichbar.",
   "reposAdmin.unstagedFilesMessage": "In Ihrem Repository befinden sich nicht bereitgestellte Dateien.",
   "resetView.resetPasswordConfirmFieldPlaceholderLabel": "Passwort bestätigen",
   "resetView.resetPasswordError": "Fehler beim Zurücksetzen des Passworts. Token ist möglicherweise ungültig oder abgelaufen.",
   "resetView.resetPasswordFieldPlaceholderLabel": "Neues Kennwort",
   "resetView.resetPasswordInvalidToken": "Ungültiges oder abgelaufenes Token.",
   "resetView.resetPasswordSuccess": "Passwort erfolgreich zurückgesetzt. Bitte melden Sie sich mit Ihrem neuen Passwort an.",
   "rteControlMessages.escapeScripts": "Escape-Skripte",
+  "rteControlMessages.incompatibleDatasource": "Die zum Durchsuchen konfigurierte Datenquelle ist nicht mit dem Rich-Text-Editor kompatibel. Bitte wenden Sie sich an Ihren Administrator.",
```
Comment (sumerjabri) [inline ui/app/src/translations/locales/de.json:324]: Is this related to this PR?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: ui/app/src/translations/locales/de.json (hunk 4 above)
Comment (rart) [reply ui/app/src/translations/locales/de.json:324]: Yes.

**Review by sumerjabri — APPROVED** [review]
